pandas/tests/io/parser/test_common.py

```diff
@@ -1351,6 +1351,30 @@ def test_numeric_range_too_wide(all_parsers, exp_data):
     tm.assert_frame_equal(result, expected)
 
 
+@pytest.mark.parametrize("neg_exp", [-617, -100000, -99999999999999999])
+def test_very_negative_exponent(all_parsers, neg_exp):
+    # GH#38753
```

Comment:
is this with similar tests?

Reply:
Tests above this test overflow, out-of-range numbers that should be parsed as string, edge cases on ends of feasible range, so seemed like the a good spot.

Reply:
This location makes sense for now, but we might be nearing a point where a separate module will do better.